ballerina/Ballerina.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 org = "ballerina"
 name = "data.xmldata"
-version = "1.0.0"
+version = "1.0.1"
```

Comment:
I think this should be a minor, since the runtime behaviour changes (despite it being a bug previously).

Reply:
The runtime behaviour only changes for the Finite types ryt?
And also its a error become a valid output.
So no breaking change.

Isnt is better to release a patch?
And also the next minor version will be pack with U11

Reply:
If someone tested the result to see if it `is error`, what previously returned true now returns false (or vice versa). While it is an edge case and relies on a bug, it can still be considered breaking.

Reply:
This is a valid usecase, so get the result and check it as a error is not a practical usage I guess.